diff -up redis-8.4.0/deps/Makefile.2~ redis-8.4.0/deps/Makefile
--- redis-8.4.0/deps/Makefile.2~	2025-11-18 15:58:24.000000000 +0100
+++ redis-8.4.0/deps/Makefile	2026-01-13 15:28:22.346363824 +0100
@@ -53,14 +53,6 @@ ifneq ($(shell sh -c '[ -f .make-ldflags
 endif
 
 distclean:
-	-(cd hiredis && $(MAKE) clean) > /dev/null || true
-	-(cd linenoise && $(MAKE) clean) > /dev/null || true
-	-(cd lua && $(MAKE) clean) > /dev/null || true
-	-(cd jemalloc && [ -f Makefile ] && $(MAKE) distclean) > /dev/null || true
-	-(cd hdr_histogram && $(MAKE) clean) > /dev/null || true
-	-(cd fpconv && $(MAKE) clean) > /dev/null || true
-	-(cd fast_float && $(MAKE) clean) > /dev/null || true
-	-(cd xxhash && $(MAKE) clean) > /dev/null || true
 	-(rm -f .make-*)
 
 .PHONY: distclean
