diff -up redis-7.2.0/deps/Makefile.2~ redis-7.2.0/deps/Makefile
--- redis-7.2.0/deps/Makefile.2~	2023-08-15 20:04:30.491314582 +0200
+++ redis-7.2.0/deps/Makefile	2023-08-15 20:05:43.965959077 +0200
@@ -36,12 +36,6 @@ ifneq ($(shell sh -c '[ -f .make-ldflags
 endif
 
 distclean:
-	-(cd hiredis && $(MAKE) clean) > /dev/null || true
-	-(cd linenoise && $(MAKE) clean) > /dev/null || true
-	-(cd lua && $(MAKE) clean) > /dev/null || true
-	-(cd jemalloc && [ -f Makefile ] && $(MAKE) distclean) > /dev/null || true
-	-(cd hdr_histogram && $(MAKE) clean) > /dev/null || true
-	-(cd fpconv && $(MAKE) clean) > /dev/null || true
 	-(rm -f .make-*)
 
 .PHONY: distclean
