--- redis-4.0.8/deps/Makefile.omv~	2018-02-06 18:32:27.706799033 +0100
+++ redis-4.0.8/deps/Makefile	2018-02-06 18:32:57.390912727 +0100
@@ -33,10 +33,6 @@ ifneq ($(shell sh -c '[ -f .make-ldflags
 endif
 
 distclean:
-	-(cd hiredis && $(MAKE) clean) > /dev/null || true
-	-(cd linenoise && $(MAKE) clean) > /dev/null || true
-	-(cd lua && $(MAKE) clean) > /dev/null || true
-	-(cd jemalloc && [ -f Makefile ] && $(MAKE) distclean) > /dev/null || true
 	-(rm -f .make-*)
 
 .PHONY: distclean
