diff --git a/runtest b/runtest
index 2ea4d39..257a55b 100755
--- a/runtest
+++ b/runtest
@@ -1,5 +1,5 @@
 #!/bin/bash
-TCL=tclsh8.5
+TCL=tclsh8.6
 which $TCL
 if [ "$?" != "0" ]
 then
diff --git a/tests/integration/replication.tcl b/tests/integration/replication.tcl
index 803d9b5..cae0d91 100644
--- a/tests/integration/replication.tcl
+++ b/tests/integration/replication.tcl
@@ -74,7 +74,7 @@ start_server {tags {"repl"}} {
 }
 
 proc start_write_load {host port seconds} {
-    exec tclsh8.5 tests/helpers/gen_write_load.tcl $host $port $seconds &
+    exec tclsh8.6 tests/helpers/gen_write_load.tcl $host $port $seconds &
 }
 
 proc stop_write_load {handle} {
diff --git a/tests/integration/replication.tcl.orig b/tests/integration/replication.tcl.orig
index 278496a..0284946 100644
--- a/tests/integration/replication.tcl.orig
+++ b/tests/integration/replication.tcl.orig
@@ -74,7 +74,7 @@ start_server {tags {"repl"}} {
 }
 
 proc start_write_load {host port seconds} {
-    exec tclsh8.5 tests/helpers/gen_write_load.tcl $host $port $seconds &
+    exec tclsh8.6 tests/helpers/gen_write_load.tcl $host $port $seconds &
 }
 
 proc stop_write_load {handle} {
diff --git a/tests/test_helper.tcl b/tests/test_helper.tcl
index d390624..e7c35c5 100644
--- a/tests/test_helper.tcl
+++ b/tests/test_helper.tcl
@@ -180,7 +180,7 @@ proc test_server_main {} {
     set start_port [expr {$::port+100}]
     for {set j 0} {$j < $::numclients} {incr j} {
         set start_port [find_available_port $start_port]
-        set p [exec tclsh8.5 [info script] {*}$::argv \
+        set p [exec tclsh8.6 [info script] {*}$::argv \
             --client $port --port $start_port &]
         lappend ::clients_pids $p
         incr start_port 10
