
https://github.com/antirez/redis/commit/459e2975f40b9d4e0ddc488d45d39e0cdd334f5c

diff -Nurp redis-2.4.8.orig/tests/integration/replication.tcl redis-2.4.8/tests/integration/replication.tcl
--- redis-2.4.8.orig/tests/integration/replication.tcl	2012-02-22 15:23:43.000000000 +0200
+++ redis-2.4.8/tests/integration/replication.tcl	2012-04-28 18:48:32.188691658 +0300
@@ -97,7 +97,7 @@ start_server {tags {"repl"}} {
                     [lindex $slaves 2] slaveof $master_host $master_port
 
                     # Wait for all the three slaves to reach the "online" state
-                    set retry 100
+                    set retry 500
                     while {$retry} {
                         set info [r -3 info]
                         if {[string match {*slave0:*,online*slave1:*,online*slave2:*,online*} $info]} {
