diff -up redis-8.2.3/deps/Makefile.2~ redis-8.2.3/deps/Makefile
--- redis-8.2.3/deps/Makefile.2~	2025-11-02 14:43:53.000000000 +0100
+++ redis-8.2.3/deps/Makefile	2025-11-03 02:28:10.571792923 +0100
@@ -53,13 +53,6 @@ ifneq ($(shell sh -c '[ -f .make-ldflags
 endif
 
 distclean:
-	-(cd hiredis && $(MAKE) clean) > /dev/null || true
-	-(cd linenoise && $(MAKE) clean) > /dev/null || true
-	-(cd lua && $(MAKE) clean) > /dev/null || true
-	-(cd jemalloc && [ -f Makefile ] && $(MAKE) distclean) > /dev/null || true
-	-(cd hdr_histogram && $(MAKE) clean) > /dev/null || true
-	-(cd fpconv && $(MAKE) clean) > /dev/null || true
-	-(cd fast_float && $(MAKE) clean) > /dev/null || true
 	-(rm -f .make-*)
 
 .PHONY: distclean
